### ISSUE TITLE
angle_3b not MPI ready

### ISSUE THREAD

**Issue by MES-physics** [libAtoms/QUIP#640]
Hello,
I have a newly installed MPI version, (OneAPI and MKL) and I get this error :
```descriptor_MPI_setup: angle_3b not MPI ready.```
when trying to run this script :

```quip E=T F=T atoms_filename=Atoms.xyz param_filename=Cpotential.xml | grep AT | sed 's/AT//' > quip_trainGAPC.xyz```
And the output file is empty. What can I do? Thanks.

**Comment by albapa** [comment]
I suggest you also compile a **non-MPI** version of QUIP, in order to evaluate your potentials. You can keep the OpenMP parallelisation.

This is the version you need to link it with LAMMPS if you want to use your potential from LAMMPS.

**Comment by MES-physics** [comment]
Yes, I've always had a serial version linked to lammps.  When I tried to evaluate with it, it told me I needed to update the version to match the .xml potential generated.
On another serial version on my laptop, trying the evaluation produces an empty file with no errors shown.
So, if I update the serial version now, will it break the lammps link, and necessitate building lammps with the link all over again? 
And is it necessary or desirable to evaluate with the serial quip version?    Thanks so much.
